app/helpers/application_helper.rb

```diff
@@ -3,9 +3,13 @@
 module ApplicationHelper
   def link_to_switch_locale
     if I18n.locale == :'en-US'
-      link_to 'JA', locale: 'ja'
+      [
+        link_to('CZ', locale: 'cs-CZ'),
+        link_to('JA', locale: 'ja'),
+        link_to('PT', locale: 'pt')
+      ]
     else
-      link_to 'EN', locale: 'en-US'
+      [link_to('EN', locale: 'en-US')]
```

Comment:
Why can you choose `CZ`, `JA`, or `PT` when current locale is `en-US`, but only `EN` when locale is `ja`, `cs-CZ`, or `pt`?

Reply:
But having all options all the time might be good too - and it would remove the need for an if statement.

Reply:
++ for allowing free exchange between any language :)

Reply:
Done.